controllers/main.go

```diff
@@ -1961,7 +1966,7 @@ func (controller *MainController) Voting(c web.C, r *http.Request) (string, int)
 		c.Env["Agenda"+strk+"Selected"] = v
 	}
 	c.Env["Admin"], _ = controller.isAdmin(c, r)
-	c.Env["Agendas"] = controller.agendas()
+	c.Env["Agendas"] = controller.agendas(nil)
```

Comment:
I don't really like this change, passing `nil` into this function is non-intuitive. On top of that, passing a function into a function is unnecessary complex for what we are trying to achieve here. The small gain in test coverage isn't worth the extra complexity imo.

Compromising the quality of production code in order to support testing code is usually a code smell. Perhaps this code needs a bit of refactoring before some tests are added (if we really even need tests in this area).

Reply:
Ok, will look into refactoring. Any ideas to make it cleaner?

Reply:
Not refactored but product code is unchanged now.